src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -275,7 +275,7 @@ message ShardWorker {
 }
 
 message WorkerChange {
-  message Add {}
+  message Add { google.protobuf.Timestamp effectiveAt = 1; }
```

Comment:
Don't inline this, give the block the same spacing as everything else.

Reply:
Formatter complains if I don't make it inline. This might be why all types with a single property are also defined inline in this file.
https://github.com/bazelbuild/bazel-buildfarm/blob/2097ef997a0d42a3c0290deb104b3660d697de1d/src/main/protobuf/build/buildfarm/v1test/buildfarm.proto#L80

Reply:
Huh, good to know we have linting protection for proto.